Sources/Extensions/SwiftStdlib/CollectionExtensions.swift

```diff
@@ -122,6 +122,21 @@ public extension Collection where Index == Int {
         return slices
     }
 
+    /// SwifterSwift: Returns the element at the specified position. If index is
+    /// negative, the nth element from the end will be returned. If index is out
+    /// of the indices, nil will be returned.
+    ///
+    ///        let arr = [1, 2, 3, 4, 5]
+    ///        arr[nth: 1] -> 2
+    ///        arr[nth: -2] -> 4
+    ///        arr[nth: -6] -> nil
+    ///
+    /// - Parameter index: The position of the element to access.
+    public subscript(nth index: Index) -> Element? {
```

Comment:
The regular `subscript` doesn't return an `Optional`, so I think it would be consistent to do the same. If it's out of bounds then let the program crash.

Reply:
@guykogus OK. I personally like javascript style, we know what developer want to do, crash makes people annoyed. But you are right, this is Swift.